src/controllers/playback/video/index.js

```diff
@@ -733,6 +734,24 @@ import { PluginType } from '../../../types/plugin.ts';
             return (currentTimeMs - programStartDateMs) / programRuntimeMs * 100;
         }
 
+        function updateRatingDisaply(itemInfo) {
```

Comment:
no need to add new function here you can use mediaInfo.getMediaInfoHtml(item, {
                officialRating: false,
                criticRating: true,
                starRating: true,
                endsAt: false,
                year: false,
                programIndicator: false,
                runtime: false,
                subtitles: false
            });

Reply:
Ill give it a try to see if I can work it in there, I originally opted for this way to have custom css classes that were specifically labeled for the osd, but I could add the mediainfo classes to the osd scss instead

Reply:
Alright,

I modified it to use mediaInfo, and adjusted the scss file to look for the mediainfo classes

Reply:
sorry will have one more as I had some css classes I could remove since they were not really being used

Reply:
yes u can use osdMediaInfo classes

Reply:
Alright It seems like everything on this commit is good, everything lines up and works properly, hides with the time when the viewport is small, and I didn't have to worry about a brand new function with slightly odd behavior ha ha
I added
```
originalAirDate: false,
episodeTitle: false
```
to the list of options as when I went to a show I had, it showed the original air date so I went through the function to find any missing options and these were it.